### PR TITLE
Enforces limit on length of the command string

### PR DESCRIPTION
**Pull request by shamsimam**
## Changes proposed in this PR

- enforces validation limit on cmd - must be no larger than 1K characters
- issues friendly error message when command string validation fails

## Why are we making these changes?

We would like to prevent users from sending huge command strings as inputs with a goal of avoiding stress on our backing store.

### CODE REVIEW

File: waiter/src/waiter/service_description.clj [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
 (def service-description-schema
   {;; Required
-   (s/required-key "cmd") schema/non-empty-string
+   (s/required-key "cmd") (s/both s/Str (s/pred #(<= 1 (count %) 1024)'at-most-1K-chars))
```
Comment (DaoWen) [inline waiter/src/waiter/service_description.clj]: Missing a space before the quoted symbol `'at-most-1K-chars`.

Comment (DaoWen) [reply waiter/src/waiter/service_description.clj]: Maybe maximum length should be configurable, but with the default of 1024.

File: waiter/test/waiter/token_test.clj [hunk 2]
```diff
@@ -1179,6 +1179,24 @@
           (is (= 400 status))
           (is (str/includes? body "Cannot modify previous token metadata"))))
 
+      (testing "post:new-service-description:bad-token-command"
+        (let [kv-store (kv/->LocalKeyValueStore (atom {}))
+              service-description (walk/stringify-keys
+                                    {:cmd (str "tc " (str/join "" (repeat 1024 "x")))
```
Comment (DaoWen) [inline waiter/test/waiter/token_test.clj]:
```suggestion
                                    {:cmd (apply str "tc " (repeat 1024 "x"))
```

File: waiter/src/waiter/service_description.clj (hunk 1 above)
Comment (DaoWen) [inline waiter/src/waiter/service_description.clj]: Add missing space:
```suggestion
   (s/required-key "cmd") (s/both s/Str (s/pred #(<= 1 (count %) 1024) 'at-most-1K-chars))
```

Comment (shamsimam) [reply waiter/src/waiter/service_description.clj]: I will introduce it as a `constraint`.

**Pull request merged**